doc/whats_new.rst

```diff
@@ -71,6 +71,8 @@ API
 
     - Adds :func:`mne.time_frequency.csd_epochs` to replace :func:`mne.time_frequency.csd_compute_epochs` for naming consistency. :func:`mne.time_frequency.csd_compute_epochs` is now deprecated and will be removed in mne 0.14, by `Nick Foti`_
 
+    - Weighted addition and subtraction of :class:`Evoked` as ``ev1 + ev2`` and ``ev1 - ev2`` have been deprecated, use explicit :func:`mne.combine_evoked` instead by `Eric Larson`_
```

Comment:
use explicit :func:`mne.combine_evoked` with weights='nave'?

but now I realize that the ev1 - ev2 is not easy to do anymore.

mne.combine_evoked([ev1, -ev2], weights='nave')

does not work. Do I have to access the nave attribute and do the maths myself to do a weighted contrast?